geoportailv3/static/js/infobar/elevationdirective.js

```diff
@@ -44,25 +44,29 @@ app.module.directive('appElevation', app.elevationDirective);
  * @ngInject
  * @constructor
  * @param {angular.$http} $http
+ * @param {angular.Scope} $scope
  * @param {ngeo.Debounce} ngeoDebounce
  * @param {string} elevationServiceUrl
  */
 app.ElevationDirectiveController =
-    function($http, ngeoDebounce, elevationServiceUrl) {
+    function($http, $scope, ngeoDebounce, elevationServiceUrl) {
   var map = this['map'];
+  this['parentCtrl'] = $scope['$parent']['ctrl'];
```

Comment:
Here you make the assumption that the parent controller name is `ctrl`. It's an assumption you should not make in my opinion. There must be a better way.